test/black-box/lib/BlackBoxTestUtil.ts

```diff
@@ -14,7 +14,8 @@ export class BlackBoxTestUtil {
      * @returns {Promise<void>}
      */
     public async cleanDb(ignoreTables: string[] = []): Promise<any> {
-        const res = await rpc('cleandb', ignoreTables);
+        ignoreTables.unshift('clean'); // push subcommand for cleadDB call
+        const res = await rpc('data', ignoreTables);
         res.expectJson();
```

Comment:
const res = await rpc('data', ignoreTables);
should be
const res = await rpc(Commands.DATA_ROOT, [Commands.DATA_CLEAN, ignoreTables]);

Reply:
or rename the ignoreTables if it contains multiple params